setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# (C) 2019 எழில் மொழி அறக்கட்டளை
+# பைந்தமிழ் நிரல் தொகுப்பிற்கு எங்களது பங்களிப்பு
+#
```

Comment:
I would request you to remove the copyright headers for this reason https://opensource.com/law/14/2/copyright-statements-source-files.

Also please share if there is any steps to compile the setup.py

Reply:
NP.

Reply:
$ python3 setup.py sdist build
அப்படின்னு கொடுத்தா மட்டும் போதும். வேற எந்த மாதிரியும் கம்பைல் பன்ன தேவை இல்லை.
 
நீங்க சொன்ன வேலைகளை திருத்திடலாம் :)